app.js

```diff
@@ -24,82 +24,93 @@ class Results extends React.Component {
     } else {
       newList = currentList;
     }
-    this.setState({
-      results: newList
-    });
+
+    // this.setState({
```

Comment:
Just commented this out as I wasnt actually sure why you wanted to set this state here, but if you need to change the results state from the parent component you will want to pass a callback function down to this component as a prop which sets the sate property on the parent component.